connector-packager/tests/test_xsd_validator.py

```diff
@@ -131,3 +129,26 @@ def test_warn_defaultSQLDialect_as_base(self):
         self.assertEqual(len(cm.output), 1)
         self.assertIn('DefaultSQLDialect', cm.output[0], "DefaultSQLDialect not found in warning message")
 
+    def test_warn_authentication_attribute(self):
```

Comment:
Can you also test that when attribute-list isn't specified no warning is emitted.  This occurs in some inherited cases and inferred connection resolver.